docs/future/core/reference/corectl.md

```diff
@@ -228,6 +231,52 @@ Argument:
 
 * **address**: The listen address, in `host:port` format, to be added to the allowed members list.
 
+
+### `get`
+
+Retrieves the current value of a configuration option.
+
+```
+corectl get [key]
+```
+
+Argument:
+
+* **key**: The configuration option to retrieve.
+
+
+### `add`
+
+Adds a tuple to a configuration option's set of values. The configuration option must be defined as a set. The number of value arguments provided must match the configuration option's tuple length. Add will error if a conflicting tuple already exists in the set. To overwrite the existing tuple, provide the `-u` flag.
+
+```
+corectl add [key] [value]...
+```
+
+Flags:
+
+* **-u**: Overwrite the existing tuple if it conflicts.
```

Comment:
Grammar nit (sorry 😞 ): sometimes the verbs are in imperative mood ("Overwrite", "Remove"), and in other cases they're not ("Adds", "Retrieves").